acceptance/acceptance_test.go

```diff
@@ -178,6 +187,86 @@ func thereShouldBeNoWarningsInTheResult(ctx context.Context) error {
 	return nil
 }
 
+func opaEval(ctx context.Context, evalString string) (context.Context, error) {
+	ts, err := getTestState(ctx)
+	if err != nil {
+		return ctx, fmt.Errorf("opaEval get test state: %w", err)
+	}
+
+	cmd := exec.Command(
+		"go",
+		"run",
+		"github.com/enterprise-contract/ec-cli",
+		"opa",
+		"eval",
+		"--data",
+		"./policy", // everything
+		"--input",
+		ts.inputFileName,
+		evalString,
+	)
+	cmd.Dir = ts.variables["GITROOT"]
+
+	var stdout bytes.Buffer
+	var stderr bytes.Buffer
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+
+	if err := cmd.Run(); err != nil {
+		return ctx, fmt.Errorf("running ec opa eval: %w\n%s", err, stderr.String())
+	}
+
+	ts.stdout = string(stdout.Bytes())
+
+	return setTestState(ctx, ts), nil
+}
+
+func theResultShouldBe(ctx context.Context, expectedResult string) error {
+	ts, err := getTestState(ctx)
+	if err != nil {
+		return fmt.Errorf("reading test state: %w", err)
+	}
+
+	// Unmarshal the expected result from json so we don't need to worry
+	// about formatting diffs
+	var expectedResultData any
+	err = json.Unmarshal([]byte(expectedResult), &expectedResultData)
+	if err != nil {
+		return fmt.Errorf("parsing expected result json: %w", err)
+	}
+
+	// Marshal it back to consistently indented json
+	// Todo maybe: use github.com/yudai/gojsondiff to produce diffs
+	expected, err := json.MarshalIndent(expectedResultData, "", "  ")
+	if err != nil {
+		return fmt.Errorf("remarshalling expected result data: %w", err)
+	}
+
+	// Unmarshal the actual opa eval result
+	// We get something like this on stdout:
+	//   {"result":[{"expressions":[{"value": ...}]}]}
+	// I think we'll generally have just have one expression and one result,
+	// so let's pull that value up so the feature files are a little tidier
+	var parsedOutput map[string][]map[string][]map[string]any
```

Comment:
Maybe just use `var parsedOutput any` here for consistency with the expected result?